kiwi/bootloader/template/grub2.py

```diff
@@ -73,8 +73,8 @@ def __init__(self):
         ''').strip() + os.linesep
 
         self.header_terminal_setup = dedent('''
-            terminal_input ${terminal_setup}
-            terminal_output ${terminal_setup}
+            terminal_input ${terminal_input}
```

Comment:
This breaks if `terminal_input` or `terminal_output` are empty

Reply:
If not set this leads to the following settings in the template
```
...
set timeout=10
set timeout_style=menu
terminal_input 
terminal_output 
menuentry ...
```

According to the grub docs this leads to the default settings for in and output. In my tests it does that and did not cause an issue. Do you have different experience ?